CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Jupyter integration: Visual rendering for data cubes (process graphs) shown instead of a object information.  ([#336](https://github.com/Open-EO/openeo-python-client/issues/336))
```

Comment:
```suggestion
- Jupyter integration: show process graph visualization of `DataCube` objects instead of generic `repr`.  ([#336](https://github.com/Open-EO/openeo-python-client/issues/336))
```